packages/apps/postgres/README.md

```diff
@@ -66,44 +66,57 @@ See:
 
 ### Common parameters
 
-| Name              | Description                                                                                                                             | Value   |
-| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `replicas`        | Number of Postgres replicas                                                                                                             | `2`     |
-| `resources`       | Explicit CPU and memory configuration for each PostgreSQL replica. When left empty, the preset defined in `resourcesPreset` is applied. | `{}`    |
-| `resourcesPreset` | Default sizing preset used when `resources` is omitted. Allowed values: nano, micro, small, medium, large, xlarge, 2xlarge.             | `micro` |
-| `size`            | Persistent Volume size                                                                                                                  | `10Gi`  |
-| `storageClass`    | StorageClass used to store the data                                                                                                     | `""`    |
-| `external`        | Enable external access from outside the cluster                                                                                         | `false` |
+| Name               | Description                                             | Type        | Value   |
+| ------------------ | ------------------------------------------------------- | ----------- | ------- |
+| `replicas`         | Number of Postgres replicas                             | `int`       | `2`     |
+| `resources`        | Resources                                               | `object`    | `{}`    |
+| `resources.cpu`    | CPU                                                     | `*quantity` |         |
+| `resources.memory` | Memory                                                  | `*quantity` |         |
+| `resourcesPreset`  | Default sizing preset used when `resources` is omitted. | `string`    | `micro` |
+| `size`             | Persistent Volume size                                  | `string`    | `10Gi`  |
+| `storageClass`     | StorageClass used to store the data                     | `string`    | ``      |
```

Comment:
Empty string value should be "", not ``. And storage class can't be null, it has a system default.